Formula/n/neovim.rb

```diff
@@ -119,11 +121,13 @@ def install
     # Replace `-dirty` suffix in `--version` output with `-Homebrew`.
     inreplace "cmake/GenerateVersion.cmake", "--dirty", "--dirty=-Homebrew"
 
-    system "cmake", "-S", ".", "-B", "build",
-                    "-DLUV_LIBRARY=#{Formula["luv"].opt_lib/shared_library("libluv")}",
-                    "-DLIBUV_LIBRARY=#{Formula["libuv"].opt_lib/shared_library("libuv")}",
-                    "-DLPEG_LIBRARY=#{Formula["lpeg"].opt_lib/shared_library("liblpeg")}",
-                    *std_cmake_args
+    args = [
+      "-DLUV_LIBRARY=#{Formula["luv"].opt_lib/shared_library("libluv")}",
+      "-DLIBUV_LIBRARY=#{Formula["libuv"].opt_lib/shared_library("libuv")}",
+      "-DLPEG_LIBRARY=#{Formula["lpeg"].opt_lib/shared_library("liblpeg")}",
+    ]
+    args << "-DENABLE_WASMTIME=ON" if build.head?
+    system "cmake", "-S", ".", "-B", "build", *args, *std_cmake_args
```

Comment:
No, please don't do that. This is intentionally not enabled by default yet.

Reply:
🤷 you already maintain that `--HEAD` is broken, what's a little more breakage?

Reply:
No, _you_ maintain that; I'm just passing it on.

It's simply not sufficiently tested yet (for lack of WASM parsers in the wild), and we -- and I'm sure you -- would prefer if testing was first done by people who know what they're doing. We'll graduate it soon enough (possibly 0.11 already, if the timing works out), so it's good to know that homebrew can and will enable it in their builds. So ❤️ for that!

Reply:
I don't think I've ever said `--HEAD` was broken. (It's my daily driver, after all -- and I rebuild it several times a week.) I do say it's _unsupported_. (In theory at least. My rebuilding it relatively often means it works if you know the right knobs to turn.)

Reply:
(And, yes, having homebrew HEAD builds not be completely broken is one of the goals of WASM parser support; but there's a few more moving parts that need to be added for that.)

Reply:
> I don't think I've ever said --HEAD was broken.  (It's my daily driver, after all -- and I rebuild it several times a week.) I do say it's unsupported.

Well, fair. It _is_ broken by our definition though; that you know how to work around it says more about you than it does about the build... (There's enough people who simply `brew install --HEAD` but _don't_, and those need a clear message.)

Reply:
Never really got why having to install nvim-treesitter was a big deal. But, fine, let's fix that.

`brew install --HEAD neovim` will now install the parser versions encoded in your `cmake.deps/deps.txt` file. The parser names are still hardcoded (and if I have sufficient motivation one day I might fix that), but it doesn't seem like there's too much churn for those upstream.

Reply:
We don't envision expanding (or shrinking) that list in the foreseeable future (except possibly if the markdown parser ever returns to a "single parser" model).

Reply:
Sufficient motivation came early: https://github.com/Homebrew/homebrew-core/commit/1de1d102474285c1f487861c75cd46bb04cbaea9